### ISSUE TITLE
ps3netsrv "buffer overflow detected" When loading game

### ISSUE THREAD

**Issue by Faithy3** [aldostools/webMAN-MOD#1123]
Hi there! 
I have a server running ps3netsrv for my PS3. using webMAN, I can load games from hard drive just fine, but loading any game from network causes the following output in terminal, followed by the program exiting:
```ps3netsrv build 20240709 (mod by aldostools)
Path: /srv/Data/Roms/PS3

Host IP #1: 192.168.1.118:38008
Waiting for client...
[0] Connection from 192.168.1.81
open dir /GAMES
building virtual iso...
*** buffer overflow detected ***: terminated
Aborted (core dumped)
```

Server: Arch Linux (6.6.52-lts, GCC 14.2.1) running ps3netsrv build 20240709
PS3: webMAN MOD 1.47.46h on Evilnat 4.91.2 Cobra 8.5 Beta10 PEX

All of my games are ripped as folders, I don't have any ISOs to test at the moment.

Any help would be appreciated, thanks a bunch!

**Comment by Faithy3** [comment]
Building ps3netsrv myself fixed my problem. (I only just found the readme for ps3netsrv itself now, my bad)

**Comment by jedcooper** [comment]
> Building ps3netsrv myself fixed my problem. (I only just found the readme for ps3netsrv itself now, my bad)

Yes, I'm on Arch Linux and successfully built the binary. I've cloned this whole .git to my ~/git and in _Projects_/ps3netsrv I ran the ./Make.sh resulting in a ps3netsrv binary.
Copied that sudo to /usr/local/bin and short tests reveal: no buffer overflow issues anymore so far.
OFC you'd have to uninstall any previous builds or installs of the AUR.
EDIT: PS3ISO also work as well
